apps/client/src/routes/Main/Main.tsx

```diff
@@ -101,7 +101,6 @@ export const Main = () => {
               }}
             >
               <BookingInfo
-                vpoReferenceNumber={user.vpoReferenceNumber}
```

Comment:
Can you elaborate on the reason of removing this line?

Reply:
This prop wasn't used inside the component.